packages/flutter_tools/test/commands.shard/hermetic/precache_test.dart

```diff
@@ -41,8 +33,7 @@ void main() {
     );
     await createTestCommandRunner(command).run(const <String>['precache']);
 
-    // Do not throw StateError, lock is acquired.
-    expect(() => cache.checkLockAcquired(), returnsNormally);
```

Comment:
This was testing a mocked cache, which returned null for this method and would have never failed